stripe/src/main/java/com/stripe/android/view/CardMultilineWidget.java

```diff
@@ -172,6 +172,25 @@ public void setShouldShowPostalCode(boolean shouldShowPostalCode) {
         adjustViewForPostalCodeAttribute();
     }
 
+    /**
+     * Set the card number. Method does not change text field focus.
+     *
+     * @param cardNumber card number to be set
+     */
+    public void setCardNumber(String cardNumber) {
```

Comment:
Can you add tests for the various types of card numbers (e.g. cards with spaces, cards without?)

Reply:
Added them. Though, TBH, I don't think adding tests here do adds that much safety. More test is better than fewer and this doesn't feel like a fragile test so I'm in the it can't hurt camp. 

This is a one line wrapper file that exists to expose existing functionality to users on the SDK. That existing functionality is already tested
https://github.com/stripe/stripe-android/blob/6670d21b18904b3adf53a3fe1a1ca897b847efc8/stripe/src/test/java/com/stripe/android/view/CardMultilineWidgetTest.java#L115

Reply:
Also one more nit, might be good practice to annotate new methods we add (`@NonNull` in this case)